pom.xml

```diff
@@ -368,12 +369,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-branch-source</artifactId>
-            <version>2.2.3</version>
+            <version>2.3.2</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-multibranch</artifactId>
-            <version>2.16</version>
+            <version>${workflow.support.version}</version>
```

Comment:
Is this always going to be the same as `workflow-support`?

Reply:
Yeah this doesn't seem right. They are different plugins and can have different versions.

Reply:
That was my IDE being shit. Sorry about that.